### PR TITLE
Update CHANGELOG.md for v1.10.0

### PR DESCRIPTION
**Pull request by arnaldo2792**
**Description of changes:**
Adds change log for v1.10.0

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Comment by arnaldo2792** [comment]
(Forced push adds admin and control container PRs)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,141 @@
+# v1.10.0 (2022-10-10)
+
+## OS Changes
+* Automated reconciliation of boot settings ([#2375])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Add optional setting to reboot into new kernel command line parameters ([#2375])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,141 @@
...
+* Support for static IP addressing ([#2204], [#2330], [#2445])
+* Add sources for NVIDIA's 515 driver ([#2455])
```
Comment (bcressey) [inline CHANGELOG.md]: Can you find a better way to phrase this? We certainly don't have access to NVIDIA's sources.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,141 @@
...
+* Set mode for tmpfs mounts ([#2473])
+* Increase inotify default limits ([#2335])
+* Increase `vm.max_map_count` to match AL2's configurations ([#2344])
```
Comment (bcressey) [inline CHANGELOG.md]: maybe something like:
```suggestion
* Align `vm.max_map_count` with the EKS Optimized AMI ([#2344])
```

**Review by jpmcb — APPROVED** [review]
✏️

**Review by stmcginnis — APPROVED** [review]
Everything looks reasonable to me.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,141 @@
...
+* Add support to configure DNS settings through the API ([#2353])
```
Comment (zmrow) [inline CHANGELOG.md]: This might read better as something like:
"Add support for configuring DNS settings..."

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,141 @@
...
+### AWS
+* Disable drivers for USB-attached network interfaces ([#2328])
+
+### Metal
+* Add driver support for Solarflare, Pensando, Myricom, Huawei, Emulex, Chelsio, Broadcom, AMD and Intel 10G+ network cards ([#2379])
+
+## Build Changes
+* Extend `external-files` to vendor go modules ([#2378], [#2403], [#2430])
+* Make `net_config` unit tests reusable across versions ([#2385])
+* Add diff-kernel-config to identify kernel config changes ([#2368])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Add `diff-kernel-config` to identify kernel config changes ([#2368])
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,141 @@
...
+* Honor `BUILDSYS_ARCH` and `BUILDSYS_VARIANT` env variables when set ([#2425])
+* Use architecture specific json payloads in unit tests ([#2367], [#2363])
+* Add unified `check` target in `Makefile.toml` for review readiness ([#2384])
+* Add `cargo make` tasks for testing ECS variants ([#2348])
+* Add `cargo make` tasks to perform migration testing in AWS K8s ([#2273])
+* Update Go dependencies of fist-party go projects ([#2424], [#2440], [#2450], [#2452], [#2456])
+* Update Rust dependencies ([#2458])
+* Update third-party packages ([#2397], [#2398], [#2464], [#2465], thanks @kschumy)
+* Update Bottlerocket SDK to 0.27.0 ([#2428])
+* Update AWS SDK Rust ([#2415], [#2476])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Migrate `pubsys` to the AWS SDK for Rust ([#2415], [#2476])
```

#2476 can go into the "Update Rust dependencies" line.

**Review by ecpullen — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,141 @@
...
+* Update `testsys` dependencies ([#2392])
+* Fix `hotdog`'s spec URL to the correct upstream link ([#2326])
+* Fix clippy warnings and enable lints on pull requests ([#2337], [#2346], [#2443])
+* Format issue field in PR template ([#2314])
+
+## Documentation Changes
+* Update checksum for new `root.json` ([#2405])
+* Mention that boot settings are available in k8s-1.23 variants ([#2358])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Mention that boot settings are available in Kubernetes 1.23 variants ([#2358])
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,141 @@
...
+* Mention the need of AWS credentials in BUILDING.md and PUBLISHING-AWS.md ([#2334])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Mention the need for AWS credentials in BUILDING.md and PUBLISHING-AWS.md ([#2334])
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,141 @@
...
+* Migrate `netdog` from `serde_xml_rs` to `quick-xml` ([#2311])
+* Support versioning for `net.toml` ([#2281])
+* Update admin and control container ([#2471], [#2472])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Add support for Kubernetes 1.24 variants ([#2437])
+* Remove variant aws-k8s-1.19 ([#2316])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Remove Kubernetes 1.19 variants ([#2316])
```

I know there was just the one, but for consistency with future add/remove news.

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,141 @@
...
+* Add diff-kernel-config to identify kernel config changes ([#2368])
+* Extended support for variants in buildsys ([#2339])
+* Clarify crossbeam license ([#2447])
+* Switch back to `simplelog` in `pubsys` ([#2454])
+* Honor `BUILDSYS_ARCH` and `BUILDSYS_VARIANT` env variables when set ([#2425])
+* Use architecture specific json payloads in unit tests ([#2367], [#2363])
+* Add unified `check` target in `Makefile.toml` for review readiness ([#2384])
+* Add `cargo make` tasks for testing ECS variants ([#2348])
+* Add `cargo make` tasks to perform migration testing in AWS K8s ([#2273])
+* Update Go dependencies of fist-party go projects ([#2424], [#2440], [#2450], [#2452], [#2456])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Update Go dependencies of first-party go projects ([#2424], [#2440], [#2450], [#2452], [#2456])
```

👊🏻

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,141 @@
...
+### Metal
+* Add driver support for Solarflare, Pensando, Myricom, Huawei, Emulex, Chelsio, Broadcom, AMD and Intel 10G+ network cards ([#2379])
+
+## Build Changes
+* Extend `external-files` to vendor go modules ([#2378], [#2403], [#2430])
+* Make `net_config` unit tests reusable across versions ([#2385])
+* Add diff-kernel-config to identify kernel config changes ([#2368])
+* Extended support for variants in buildsys ([#2339])
+* Clarify crossbeam license ([#2447])
+* Switch back to `simplelog` in `pubsys` ([#2454])
```
Comment (bcressey) [inline CHANGELOG.md]: Let's roll this into a changelog item about moving pubsys from Rusoto to AWS SDK for Rust.

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,141 @@
...
+* Honor `BUILDSYS_ARCH` and `BUILDSYS_VARIANT` env variables when set ([#2425])
+* Use architecture specific json payloads in unit tests ([#2367], [#2363])
+* Add unified `check` target in `Makefile.toml` for review readiness ([#2384])
+* Add `cargo make` tasks for testing ECS variants ([#2348])
+* Add `cargo make` tasks to perform migration testing in AWS K8s ([#2273])
```
Comment (bcressey) [inline CHANGELOG.md]: I'd like to move this under the Kubernetes news, and the ECS ones to the ECS section.

```suggestion
* Add `cargo make` tasks to perform migration testing for Kubernetes variants in AWS ([#2273])
```

**Comment by arnaldo2792** [comment]
(Forced push addresses comment on the DNS settings item)

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,141 @@
...
+## Documentation Changes
+* Update checksum for new `root.json` ([#2405])
+* Mention that boot settings are available in k8s-1.23 variants ([#2358])
+* Mention the need of AWS credentials in BUILDING.md and PUBLISHING-AWS.md ([#2334])
+* Add China to supported regions lists ([#2315])
+* Add community section to README.md ([#2305], [#2383])
+* Standardize `userdata.toml` as the filename used in different docs ([#2446])
+* Remove commit from image name in PROVISIONING-METAL.md ([#2312])
+* Add note to CONTRIBUTING.md that outlines filenames' casing ([#2306])
+* Fix typos in `Makefile.toml`, QUICKSTART-ECS.md, QUICKSTART-EKS.md, `netdog` and `prairiedog` ([#2318])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Fix typos in `Makefile.toml`, QUICKSTART-ECS.md, QUICKSTART-EKS.md, `netdog` and `prairiedog` ([#2318], thanks @kianmeng)
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,141 @@
...
+* Fix casing for GitHub and VMware in CHANGELOG.md ([#2329])
+* Fix typo in test setup command ([#2477])
+* Fix TESTING.md link typo ([#2438])
+* Fix positional `fetch-license` argument ([#2457])
+
+[#2375]: https://github.com/bottlerocket-os/bottlerocket/pull/2375
+[#2204]: https://github.com/bottlerocket-os/bottlerocket/pull/2204
+[#2330]: https://github.com/bottlerocket-os/bottlerocket/pull/2330
+[#2445]: https://github.com/bottlerocket-os/bottlerocket/pull/2445
```
Comment (bcressey) [inline CHANGELOG.md]: nit: I would recommend sorting this list so it's easier to scan visually for duplicates.

**Comment by arnaldo2792** [comment]
(Forced push addresses comments above)

**Review by jpculp — APPROVED** [review]

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,3 +1,143 @@
...
+
+## Orchestrator Changes
+
+### ECS
+* Add `cargo make` tasks for testing ECS variants ([#2348])
+
+### Kubernetes
+
+* Add support for Kubernetes 1.24 variants ([#2437])
+* Remove Kubernetes aws-k8s-1.19 variants ([#2316])
```
Comment (bcressey) [inline CHANGELOG.md:23]: nit:
```suggestion
* Remove Kubernetes 1.19 variants ([#2316])
```

**Review by bcressey — APPROVED** [review]

**Pull request merged**